Table panel: Add alt and title text options to image cell type (#89930)
grafana/grafana: docs/sources/panels-visualizations/visualizations/table/index.md

Cells can be displayed as a graphical gauge, with several different presentation types.

#### Image

> 

Reviewer: Can you add a basic description of this cell type.

> If you want to set the alternative text of an image, use the **Alt text** option to do so which will make alternative text avaiable for screen readers and cases when images can't be loaded.

Reviewer: ```suggestion
Use the **Alt text** option to set the alternative text of an image. The text will be available for screen readers and in cases when images can't be loaded.
```

> Additionally, if you'd like to set the title text, use the **Title text** option to do so which will make the title text display when the image is hovered with a cursor.

Reviewer: ```suggestion
Use the **Title text** option to set the text that's displayed when the image is hovered over with a cursor.
```
